fury/actor.py

```diff
@@ -1759,6 +1759,86 @@ def cone(centers, directions, colors, heights=1., resolution=10,
     return actor
 
 
+def octagonalprism(centers, directions=(1, 0, 0), colors=(255, 0, 0),
+                   scale=1):
+    """Visualize one or many octagonal prisms with different features.
+
+    Parameters
+    ----------
+    centers : ndarray, shape (N, 3)
+        Octagonal prism positions
+    directions : ndarray, shape (N, 3)
+        The orientation vector of the octagonal prism.
+    colors : ndarray (N,3) or (N, 4) or tuple (3,) or tuple (4,)
+        RGB or RGBA (for opacity) R, G, B and A should be at the range [0, 1]
+    scale : int or ndarray (N,3) or tuple (3,), optional
+        Octagonal prism size on each direction (x, y), default(1)
+
+    Returns
+    -------
+    vtkActor
+
+    Examples
+    --------
+    >>> from fury import window, actor
+    >>> scene = window.Scene()
+    >>> centers = np.random.rand(3, 3)
+    >>> dirs = np.random.rand(3, 3)
+    >>> colors = np.random.rand(3, 3)*255
+    >>> scales = np.random.rand(4, 1)
+    >>> actor = actor.octagonalprism(centers, dirs, colors, scales)
+    >>> scene.add(actor)
+    >>> # window.show(scene)
```

Comment:
Hello @mlraglin , 
Executing the following example gives the following error in `fury/primitive.py`:
```python
scale = scale.reshape((big_vertices.shape[0], 1))
ValueError: cannot reshape array of size 64 into shape (48,1)
```